web/package.json

```diff
@@ -6,6 +6,7 @@
     "axios": "^0.19.0",
     "bootstrap": "^4.3.1",
     "copy-to-clipboard": "^3.2.0",
+    "dayjs": "^1.8.16",
```

Comment:
It looks like you've included dayjs, probably to replace moment & have a smaller bundle size.
It's a good idea, however moment is still a dependency and still used everywhere else.

Could you please either swap moment for dayjs in this / another PR or remove the dep?

Thank you!

Reply:
I tried to replace moment. Sadly DayJS haven’t the very useful “fromNow” method. 
Looks like I forgot to uninstall dayjs, nice shot. Gonna remove it in a next commit .

Reply:
It seems to exists under the name 'relativeTime': https://github.com/iamkun/dayjs/blob/master/docs/en/Plugin.md#relativetime

Reply:
Done. Please check new commit.